recipes/gherkin-official/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "gherkin-official" %}
+{% set version = "4.1.3" %}
+{% set sha256sum = "7f6fa7a3ceffda8bba0f56b42140cc6bae001d010fef336b5b2602252242395d" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256sum }}
+
+build:
+  number: 0
+  #script: python setup.py install --single-version-externally-managed --record=record.txt
+  script:
+    - rm gherkin/count_symbols_py2.py  # [py3k][not win]
+    - del gherkin\count_symbols_py2.py  # [py3k][win]
+    - python setup.py install
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - nose
+
+test:
+  imports:
+    - gherkin
+
+about:
+  home: https://github.com/cucumber/gherkin-python
+  license: MIT
```

Comment:
Please ask upstream to include this in the `MANIFEST` and xref the issue/PR in a comment in the `meta.yaml`.